lib/services/ngsiParser.js

```diff
@@ -211,10 +211,12 @@ function getErrorField(body) {
     var errorField = body.errorCode ||
         body.orionError;
 
-    if (body && body.contextResponses &&
-        body.contextResponses[0] && body.contextResponses[0] &&
-        body.contextResponses[0].statusCode && body.contextResponses[0].statusCode.code !== '200') {
-        errorField = body.contextResponses[0].statusCode;
+    if (body && body.contextResponses) {
+        for (var i in body.contextResponses) {
+            if (body.contextResponses[i].statusCode && body.contextResponses[i].statusCode.code !== '200') {
```

Comment:
Ups, sorry! Maybe I confused you :) I would suggest:
```javascript
if (body.contextResponses[i] && body.contextResponses[i].statusCode && body.contextResponses[i].statusCode.code !== '200') {
```
(notice the `body.contextResponses[i]` check) to protect the code against the case of an empty contextResponses array :)